charts/opentelemetry-kube-stack/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-kube-stack
-version: 0.3.0
+version: 0.3.1
```

Comment:
it looks like the linter is still failing because of some whitespacing issues? Did your linter/formatter automatically remove extra spacing potentially?

Reply:
if not, i can try running this locally and see what happens.

Reply:
@jaronoff97  I'm closing this PR and created another one https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1388